apps/challenges/challenge_notification_util.py

```diff
@@ -56,3 +56,21 @@ def construct_and_send_eks_cluster_creation_mail(challenge):
         template_id=template_id,
         template_data=template_data,
     )
+
+
+def construct_and_send_remote_evaluation_mail(challenge):
+    if settings.DEBUG:
+        return
+
+    template_data = {"CHALLENGE_PK": challenge.id,
```

Comment:
@burnerlee we also need to send `CHALLENGE_NAME`, `CHALLENGE_IMAGE_URL` and `CHALLENGE_URL` to the email template